services/github-service/src/main/java/org/kontinuity/catapult/service/github/api/NoSuchWebhookEventException.java

```diff
@@ -0,0 +1,16 @@
+package org.kontinuity.catapult.service.github.api;
+
+/**
+ * Indicates a specified Webhook event does not exist.
+ * 
+ * @author <a href="mailto:rmartine@redhat.com">Ricardo Martinelli de Oliveira</a>
+ */
+public class NoSuchWebhookEventException extends RuntimeException {
```

Comment:
Is this a result of input from the user?  If so maybe extends IllegalArgumentException?